src/coinbase/address/external_address.ts

```diff
@@ -78,7 +78,7 @@ export class ExternalAddress extends Address {
   public async stakeableBalance(
     asset_id: string,
     options: CoinbaseExternalAddressStakeOptions = { mode: StakeOptionsMode.DEFAULT },
-  ): Promise<string> {
+  ): Promise<Decimal> {
     const balances = await this.getStakingBalances(asset_id, options);
     return balances["stakeableBalance"];
```

Comment:
Curious why we don't just use `balances.stakeableBalance` here and for each of those?

Reply:
Yeah oddly enough, my IDE was not showing that as an option so defaulted to accessing the key name directly. But didn't run into issue calling it the way suggested. I updated it for all the balance types.

For future reference is there a best practice here on accessing the key?